cmd/state/main.go

```diff
@@ -201,3 +208,12 @@ func argsHaveVerbose(args []string) bool {
 	}
 	return false
 }
+
+func argsHaveNonInteractive(args []string) bool {
```

Comment:
Let's use flags.parser for this.

Reply:
Maybe not in this PR, but should we do the same for the `argsHaveVerbose` flag?

Reply:
Should I add this for this PR?